File: C++/13.cpp
#include <iostream>

using namespace std;

int main()
{

    int size;

    cout<<"Enter the number of elements: ";
    cin >> size;

    int list[size], sumOfElements = 0;

    cout << "Enter elements of the lis: " << endl;

    for (int i = 0; i < size; i++)
        cin >> list[i];

    for (int i = 0; i < size; i++)
        sumOfElements = sumOfElements + list[i];

    cout << "Sum of the elements : " << sumOfElements;
}

File: Python/13.py
myList = []
sumOfElements = 0

numElements = int(input("How many elements would you like to add? "))

for i in range(numElements):
    element = int(input("Enter an element: "))
    myList.append(element)

for element in myList:
    sumOfElements += element

print("The sum of elements is: ", sumOfElements)